source/_components/google_pubsub.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: "Google Pub/Sub"
+description: "Setup for Google Pub/Sub integration"
+date: 2019-01-12 13:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: google-pubsub.png
+ha_category: Component
```

Comment:
This is not a correct category.
Once you have adjusted this, the PR can be merged.